test/workers/backend_metric_worker_test.rb

```diff
@@ -1,77 +0,0 @@
-# frozen_string_literal: true
-
-require 'test_helper'
-
-class BackendMetricWorkerTest < ActiveSupport::TestCase
```

Comment:
I won't remove this for real, it is just temporarily to see what happens in CircleCI and I am rewriting it locally :wink: